lte/gateway/python/magma/enodebd/devices/baicells_436Q.py

```diff
@@ -298,14 +303,6 @@ def _calc_earfcn(low_freq_hz: int, high_freq_hz: int) -> int:
     return int(earfcn)
 
 
-def _calc_rsp(bandwidth_mhz: float, eirp_dbm_mhz: float) -> str:
-    log_ptx = eirp_dbm_mhz + 10 * log10(bandwidth_mhz)  # TODO Antenna Gain
-    rsp = log_ptx - 10 * log10(60 * bandwidth_mhz) + 10 * log10(1 + DEFAULT_PB)
```

Comment:
It looks like log10 and floor can now be deleted from imports

Reply:
Fixed